ground_station/src/utils.hpp

```diff
@@ -8,6 +8,8 @@
 
 #pragma once
 
+#include "telemetry/telemetry.hpp"
```

Comment:
just forward-declare the telemetry struct/class

below all includes:

 struct Telemetry; (if struct)
 class Telemetry; (if class)